components/frontend/staging/production-promotion.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Provider
+metadata:
+  name: github
+  namespace: frontend
+spec:
+  type: githubdispatch
+  address: https://github.com/controlplaneio-fluxcd/d1-apps
+  secretRef:
+    name: flux-apps
+---
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Alert
+metadata:
+  name: production-promotion
+  namespace: frontend
+spec:
+  providerRef:
+    name: github
+  summary: "Trigger promotion"
+  eventMetadata:
+    env: staging
+    cluster: staging-1
```

Comment:
This should be set with Flux env vars, the app team should't have to deal with cluster naming.